src/client/Microsoft.Identity.Client/Extensibility/AcquireTokenForClientBuilderExtensions.cs

```diff
@@ -17,18 +17,20 @@ public static class AcquireTokenForClientBuilderExtensions
         /// </summary>
         /// <param name="builder"></param>
         /// <param name="keyId">A key id to which the access token is associated. The token will not be retrieved from the cache unless the same key id is presented. Can be null.</param>
+        /// <param name="tokenType">Bearer or pop</param>
```

Comment:
We might want to write that `tokenType="pop"` will return a bare Pop token (without the signed HTTP envelope), whereas `tokenType="bearer"` will return a ready to use Pop token.

Reply:
We discussed about this offline. This parameter is used internally by MSAL to distinguish between POP and Bearer. MSAUTH_10_AT_POP returns "Bearer" tokens internally.